container/docker/handler.go

```diff
@@ -294,32 +294,8 @@ func (self *dockerContainerHandler) GetStats() (*info.ContainerStats, error) {
 }
 
 func (self *dockerContainerHandler) ListContainers(listType container.ListType) ([]info.ContainerReference, error) {
-	if self.name != "/docker" {
```

Comment:
Yes, docker will create a cgroup named `docker`, and is found as subcontainer of `/`. But `CanHandleAndAccept` will be **false** when the manager tries to get a `dockerContainerHandler` for it. Because the inspect will fail: https://github.com/google/cadvisor/blob/master/container/docker/factory.go#L147

You can also see this happening in the Docker daemon logs:
```log
ERRO[437132] Handler for GET /containers/{name:.*}/json returned error: no such id: docker 
ERRO[437132] HTTP Error                                    err=no such id: docker statusCode=404
```
Only when the container `/docker/f5a39e9c98788da89c050ca45d266ce0ddc31bd0a7e3ecbc618e166c17ade17d` is found it will actually create a handler.